extensions/ql-vscode/src/queries.ts

```diff
@@ -15,6 +15,13 @@ import { promisify } from 'util';
 import { QueryHistoryItemOptions } from './query-history';
 import { isQuickQueryPath } from './quick-query';
 
+/**
+ * Maximum number of lines to put in a binary choice dialog message,
+ * to work around the fact that we can't guarantee a scrollable text
+ * box for it.
+ */
+const MAX_MESSAGE_LINES = 20;
```

Comment:
Do we know that 20 lines is going to fit on small netbook-style screens?  I think if necessary we could afford to be more conservative here, since we provide the full output in the extension log.

Reply:
I'm fine with being more conservative, like 10.